Tests/SonarQube.MSBuild.Tasks.IntegrationTests/TargetsTests/RoslynTargetsTests.cs

```diff
@@ -290,7 +291,10 @@ public void Roslyn_SetResults_ResultsFileExists()
             properties.SonarQubeTempPath = rootInputFolder;
             properties[TargetProperties.ErrorLog] = resultsFile;
 
-            ProjectRootElement projectRoot = BuildUtilities.CreateValidProjectRoot(this.TestContext, rootInputFolder, properties);
+            string projectName = "MyProject";
+            ProjectDescriptor descriptor = BuildUtilities.CreateValidProjectDescriptor(rootInputFolder, projectName + ".csproj", false);
+            ProjectRootElement projectRoot = BuildUtilities.CreateInitializedProjectRoot(this.TestContext, descriptor, properties);
+            string expectedWorkingDirectory = Path.Combine(rootInputFolder, "out") + "\\\\" + projectName;
```

Comment:
This `\\\\` is awful. @dbmeneses in the end, did you solve this problem in the targets file?